ghost/milestone-emails/lib/MilestonesEmailService.js

```diff
@@ -115,10 +117,10 @@ module.exports = class MilestonesEmailService {
 
     /**
      *
-     * @param {Array} goalValues
+     * @param {Array.<number>} goalValues
```

Comment:
I think `number[]` is preferable here ☺️

Reply:
This errors, because I am returning a number. Returning the value of the first index of the filtered Array 😅